_posts/2021-08-25-nlog-5-0-preview1-ready.md

```diff
@@ -187,6 +187,31 @@ Having the logging-layer making calls into the dependency-injection-layer can ca
 or deadlocks. Both issues are extremely annoying to debug and diagnose, so one should be very careful to ensure interfaces
 are implemented with singleton-lifetime.
 
+### Aliases support for targets, layouts, layout renderers and conditions
+
+The following attributes are now allowed multiple times for a class:
```

Comment:
Maybe explain that is a feature for users that are creating their own custom NLog extensions, that they now have the option to define additional alias upfront.